pkg/drivers/bess/bess.go

```diff
@@ -38,11 +38,14 @@ const (
 	PORT      = "Port"
 	MODULE    = "Module"
 	VPORT     = "VPort"
+	PMDPORT   = "PMDPort"
 	PCAPPORT  = "PCAPPort"
+	UNIXPORT  = "UnixSocketPort"
 	PORTOUT   = "PortOut"
 	PORTINC   = "PortInc"
 	L2FORWARD = "L2Forward"
 	REPLICATE = "Replicate"
+    DROPGATE  = 8192    // This needs to be adjusted if MAX_GATES in bess changes
```

Comment:
Can you explain what this is used for?

Reply:
In BESS code it is "drop"

However, I think it is cheaper to actually wire to a SINK instead. See my other comment.